index.html

```diff
@@ -420,14 +419,22 @@
           settings</var>' <a>top-level browsing context</a>'s <a>origin</a>
         </li>
       </ol>
+      <dl class='idl' title='dictionary PermissionStorage'>
```

Comment:
I wonder if there is a way we can make it even more clear this is an internal-only structure.